constants/util.go

```diff
@@ -55,10 +55,11 @@ var (
 			DHubURL:          "docker.io/vout/megahit", // https://github.com/voutcn/megahit
 			OutputDir:        MegaHitOut,
 			AssemblyFileName: "/final.contigs.fa",
-			Comm: func() []string {
+			Comm: func(threads int) []string {
 				// NOTE: input filePath and outPath are mapped to Docker mounts during creation (slave/components/assembler/assembler.go:87)
 				return []string{
 					"-r", RawSeqIn,
+					fmt.Sprintf("-t %d", threads),
```

Comment:
You can use `"-t", threads`, instead of `fmt.Sprintf` since Docker joins the command with spaces between them.